.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,21 @@
+# Fast running checks for pull-requests
+
+name: Pull-Request Checks
+concurrency: 
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check-freeze-period:
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/verifyFreezePeriod.yml@master
+  check-merge-commits:
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/checkMergeCommits.yml@master
+  check-versions:
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/checkVersions.yml@master
+    with:
+      botName: Eclipse Equinox Bot
+      botMail: equinox-bot@eclipse.org
```

Comment:
```suggestion
      botMail: equinox-bot@eclipse.org
      extra-maven-args: '-f bundles'
```

Reply:
Applied.
For references: This is possible since https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/2782.